modules/microshift-adding-app-rpms-to-blueprint.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// microshift_running_applications/embed-microshift-apps-on-rhel-edge.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-adding-app-rpms-to-blueprint_{context}"]
+= Adding application RPMs to a blueprint
+
+.Prerequisites
+* You have root access to the host.
+* Workload or application RPMs exist in the `~/rpmbuild/RPMS` directory.
+
+To add application RPMs to a blueprint, you must create a local repository that Image Builder can use to create the ISO. With this procedure, the required container images for your workload can be pulled over the network.
+
+. Create a local RPM repository by running the following command:
++
+[source,terminal]
+----
+$ sudo createrepo ~/rpmbuild/RPMS/
+----
+
+. Give Image Builder access to the RPM repository by running the following command:
++
+[source,terminal]
+----
+$ sudo chmod a+rx ~
+----
+
+. Create the blueprint file, `repo-local-rpmbuild.toml`, with this example content:
++
+[source,terminal]
+----
+id = "local-rpm-build"
+name = "RPMs build locally"
+type = "yum-baseurl"
+url = "file:///home/builder/rpmbuild/RPMS"
```

Comment:
The path here needs to match the path in the other commands where the repo is set up and RPMs are copied. As things stand, that will only be true if the user name is `builder` so that `~` expands to `/home/builder` in the commands on lines 19 and 26 of this file and in the assemblies where the instructions to create and build the RPM.

Rather than changing all of those other locations, we could say that the user has to specify a part of the path themselves based on the user name. For example

```suggestion
url = "file://<HOME>/rpmbuild/RPMS"
```

Reply:
Or, just generic `<PATH>`